stream_buffer.c

```diff
@@ -371,6 +371,11 @@ static void prvInitialiseNewStreamBuffer( StreamBuffer_t * const pxStreamBuffer,
             pucAllocatedMemory = NULL;
         }
 
+        #if defined( __clang__ )
```

Comment:
Same suggestion as above about keeping the compiler specific changes out of the common files.

Reply:
Moved to CMakeFile.txt

Reply:
Can this be removed then?